docs/graph/overview.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Windows Community Toolkit - Graph Helpers and Controls
+author: shweaver-MSFT
+description: Authentication providers and Graph powered helpers that make it easy to work with Microsoft Graph APIs.
+keywords: uwp, wpf, netstandard, windows, community, toolkit, graph, login, authentication, provider, providers, identity
+dev_langs:
+  - csharp
+---
+
+# Windows Community Toolkit - Graph Helpers and Controls
+
+Welcome to the Windows Graph Toolkit! A sub-project of the [Windows Community Toolkit](https://aka.ms/wct) focused on [Microsoft Graph](https://developer.microsoft.com/en-us/graph/) and providing a set of Authentication providers and Graph powered Helpers/Controls that make it easy to work with Microsoft Graph APIs.
```

Comment:
We haven't officially called it the "Windows Graph Toolkit" though I do imply that with the short-link I created... We may to get buy-in on the name through our channels first.

Reply:
I've got to call it something. Any suggestions?

Reply:
For now just say "The Graph helpers and controls are a sub-project..."